src/Action/ExceptionAction.php

```diff
@@ -80,6 +83,12 @@ public function __invoke(FlattenException $exception, Request $request): Respons
         $headers['X-Content-Type-Options'] = 'nosniff';
         $headers['X-Frame-Options'] = 'deny';
 
-        return new Response($this->serializer->serialize($exception, $format['key']), $exception->getStatusCode(), $headers);
+        $response = new Response(null, $exception->getStatusCode(), $headers);
+
+        if (!$this->debug && $response->isServerError()) {
+            $exception->setMessage(Response::$statusTexts[$response->getStatusCode()]);
```

Comment:
@meyerbaptiste @soyuka , Woooh, it seems better ^^, waiting for @teohhanhui approval before modification if needed